### DISCUSSION TITLE
Megane 4 no sensor, temp f:

### DISCUSSION THREAD

**Discussion by orcf** [cedricp/ddt4all#903] [Q&A] [answered]
Hello

I try to update my rlink from v2.2 to v3.3.

Now I don't have parking sensors. Temperature, Eco button, etc.

I already downgrade to v2.2 again but still the same.

I have a odbii, ddt4all and pyren installed 

Please if someone can help thank you

**Comment by AndyIreland** [comment]
I am more into rlink1 and  have some experience on them but not on rlink2 (never had one) but if you have correct items:

Bluetooth or USB obd2 ELM327 preferably with PIC 18F25K80 chip and not chinese cheap clone
specially wired (or specially soldered) obd2 cable (so can see CAN2 /rlink through CAN socket) 
a windows laptop with a working internet signal in car (from home router or tethered up to mobile phone hotspot) 

no guarantees but if can have a look over the internet I might be able to see which setting are not filled out or disabled

if you do want me to take a look just give me a shout

**Reply by orcf** [reply]
Thank you for your help

I only have a WiFi obdII 
I gone buy a new one with usb connection.

**Reply by orcf** [reply]
Any one have a megane .5 dci that can share your config to compare with mine.

Best regards

**Reply by shrlnm** [reply]
@orcf have you tried to generate the config by the VIN?

**Reply by orcf** [reply]
How can I do it?

Can you explain?

**Reply by shrlnm** [reply, marked as answer]
@orcf 
t.me/pyren_bot

**Reply by orcf** [reply]
Try but the bot doesn't do anything

**Reply by orcf** [reply]
@shrlnm 
<img width="960" alt="radio id sem vin" src="https://github.com/cedricp/ddt4all/assets/25912533/7bf71a92-2b68-4bdf-b442-7d0dbcdd46cb">

i already have the files. but maybe a make a mistake. . 
i take a picture from pyren. 
on the pyren bot can you tell me what is the number after the vin??

**Reply by shrlnm** [reply]
@orcf The bot is working again
The number 283C22592R is a reference on autoconfiguration template in REPROG DB

**Reply by orcf** [reply]
Hi 

The bot send 2 macros
One can and other can2
I have to use both?

**Reply by orcf** [reply]
Ok
At the end I take to the car at Renault. Now all work and the version is 7.
63€ less on the account but not a new rlink
